src/Illuminate/Foundation/Auth/Access/AuthorizesRequests.php

```diff
@@ -49,6 +50,10 @@ public function authorizeForUser($user, $ability, $arguments = [])
      */
     protected function parseAbilityAndArguments($ability, $arguments)
     {
+        if ($ability instanceof BackedEnum) {
+            $ability = $ability->value;
+        }
```

Comment:
Should be able to use `Illuminate\Support\enum_value` function here.

Reply:
Done, so that means I don't have to check the instance anymore, right?